### PR TITLE
Fix upload logs for nightly [skip ci]

### PR DESCRIPTION
**Pull request by rajanashok**
Renamed the logs files for consistency.
Fixed the GSUtil log upload.

### CODE REVIEW

File: tests/nightly/nightly-kickoff.sh [hunk 1]
```diff
@@ -17,6 +17,12 @@
 echo "Removing VIC directory if present"
 rm -rf vic
 
+echo "Cleanup logs from previous run"
+rm -rf *.xml
```
Comment (anchal-agrawal) [inline tests/nightly/nightly-kickoff.sh]: `rm -rf` is fine here, but just `rm -f` will suffice since they're not directories.

**Comment by andrewtchin** [comment]
lgtm

**Comment by sgairo** [comment]
LGTM

**Comment by rajanashok** [comment]
♻️

**Pull request merged**